amee/NodeMsg[] nodes
int32[] tagNodes
int32[] rotateLeftNodes
int32[] rotateRightNodes
int32[] nextToWallNodes
